pkg/segment/query/querystatus.go

```diff
@@ -113,6 +114,7 @@ type RunningQueryState struct {
 	nodeResult               *structs.NodeResult
 	totalRecsToBeSearched    uint64
 	AllColsInAggs            map[string]struct{}
+	NewPipeLineResponse      *structs.PipeSearchResponseOuter
```

Comment:
very soon the `NewPipeLine` is going to be not new anymore. And maybe later we come up with some other new pipeline, so perhaps we should rename this var. maybe `PipeResp` or something